pkg/printers/internalversion/describe.go

```diff
@@ -1291,12 +1291,52 @@ func (d *PersistentVolumeClaimDescriber) Describe(namespace, name string, descri
 		return "", err
 	}
 
+	pc := d.Core().Pods(namespace)
+
+	mountPods, err := getMountPods(pc, pvc.Name)
+	if err != nil {
+		return "", err
+	}
+
 	events, _ := d.Core().Events(namespace).Search(legacyscheme.Scheme, pvc)
 
-	return describePersistentVolumeClaim(pvc, events)
+	return describePersistentVolumeClaim(pvc, events, mountPods)
+}
+
+func getMountPods(c coreclient.PodInterface, pvcName string) ([]api.Pod, error) {
+	nsPods, err := c.List(metav1.ListOptions{})
```

Comment:
Maybe we should set a limit as part of ListOptions in order to chunk potentially large responses? Don't feel strongly on this though cc @soltysh

Reply:
I don't see any other usages of limiting `ListOptions` for other describe operations, so I figure that it hasn't been an issue yet. I have no opinion on the matter though, if you think we should set a limit, I will.

Reply:
I don't expect that list to be that long, besides `kubectl describe` can and should be verbose. It's in the `kubectl get` where we try to save space with `more...` and similar.